### PR TITLE
[iOS]Fix the icon and title for stamps tab

### LINKED ISSUE #874: [iOS] Fix the icon and title for stamps tab
## Summary
Badges was renamed to Stamps and its tab icon was also updated.

### PR DESCRIPTION
**Pull request by Shoryu-Y**
## Issue
- close #874 

## Overview (Required)
- Change Stamps-tab title
- Change Stamps-tab icon
  - add stamp icon
  - remove badge icon
  - add filled tab icons
  - update RootView

## Links
- NA

## Screenshot (Optional if screenshot test is present or unrelated to UI)
Before | After
:--: | :--:
 | 

## Movie (Optional)

https://github.com/DroidKaigi/conference-app-2023/assets/44453803/f558808b-795d-4262-8bd4-0d3c9121c84a

### CODE REVIEW

**Comment by Shoryu-Y** [comment]
@takahirom 
I've made fixes. Please review the code.

File: app-ios/Modules/Sources/Stamps/StampsView.swift [hunk 1]
```diff
@@ -23,7 +23,7 @@ public struct StampsView: View {
                 .padding(16)
             }
             .background(AssetColors.Surface.surface.swiftUIColor)
-            .navigationTitle("Badges")
+            .navigationTitle("Stamps")
```
Comment (tkhs0604) [inline app-ios/Modules/Sources/Stamps/StampsView.swift:26]: 👍

**Review by tkhs0604 — APPROVED** [review]
Thank you updating and further improving its visual expression! Looks perfect!

File: app-ios/Modules/Sources/Navigation/RootView.swift [hunk 2]
```diff
@@ -34,8 +34,13 @@ public struct RootView: View {
                     Label {
                         Text("Timetable")
                     } icon: {
-                        Assets.Icons.timetable.swiftUIImage
-                            .renderingMode(.template)
+                        if selection == .timeline {
+                            Assets.Icons.timetable.swiftUIImage
+                                .renderingMode(/*@START_MENU_TOKEN@*/.template/*@END_MENU_TOKEN@*/)
```
[the remark this answers is not in the document]
Comment (Shoryu-Y) [reply app-ios/Modules/Sources/Navigation/RootView.swift]: Oh sorry...
I've deleted them now.

[the remark this answers is not in the document]
Comment (tkhs0604) [reply app-ios/Modules/Sources/Navigation/RootView.swift]: Thank you 🧹

**Pull request merged**